Dockerfile.dapper

```diff
@@ -1,25 +1,14 @@
-FROM ubuntu:20.04
-# FROM arm=armhf/ubuntu:20.04 arm64=arm64v8/ubuntu:20.04
+FROM registry.suse.com/bci/bci-base:15.4.27.14.56
 
 ARG DAPPER_HOST_ARCH
 ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH}
 
-RUN apt-get update && \
-    apt-get install -y gcc ca-certificates git wget curl vim less file && \
-    rm -f /bin/sh && ln -s /bin/bash /bin/sh
+RUN zypper -n install gcc vim less file docker git wget curl go
 
 ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH_arm64=arm64 GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
     GOPATH=/go PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
 
-RUN wget -O - https://dl.google.com/go/go1.16.4.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
-    go get golang.org/x/lint/golint
-
-ENV DOCKER_URL_amd64=https://get.docker.com/builds/Linux/x86_64/docker-1.10.3 \
-    DOCKER_URL_arm=https://github.com/rancher/docker/releases/download/v1.10.3-ros1/docker-1.10.3_arm \
-    DOCKER_URL_arm64=https://github.com/rancher/docker/releases/download/v1.10.3-ros1/docker-1.10.3_arm64 \
-    DOCKER_URL=DOCKER_URL_${ARCH}
-
-RUN wget -O - ${!DOCKER_URL} > /usr/bin/docker && chmod +x /usr/bin/docker
+RUN go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.2
```

Comment:
Why deviate from the default binary install like in every other project? Based on their own docs, `go get`/`go install` is discouraged (https://golangci-lint.run/usage/install/#install-from-source). This also blocks auto updating because it doesn't match our generic rules.

Reply:
The change was aligned with the current version in [master](https://github.com/rancher/rke-tools/blob/622f06badba2fa91754748346e759911914df344/Dockerfile.dapper#L15) which uses `go get`.

This has now changed to use upstream's `install.sh` instead.